webapp/channels/src/components/admin_console/access_control/modals/user_sync/user_sync_modal.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import {Modal} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import {getProfilesByIds} from 'mattermost-redux/actions/users';
+
+import {SyncedUserList} from './synced_user_list';
+
+import './user_sync_modal.scss';
+
+// Types for sync results
+export type ChannelMembersSyncResults = {
+    MembersAdded: string[];
+    MembersRemoved: string[];
+};
+
+// Modal for showing detailed user lists
+type UserListModalProps = {
+    channelId: string;
+    channelName: string;
+    syncResults: ChannelMembersSyncResults;
+    onClose: () => void;
+};
+
+export const UserListModal = ({channelId, channelName, syncResults, onClose}: UserListModalProps): JSX.Element => {
+    const [activeTab, setActiveTab] = useState<'added' | 'removed'>('added');
+
+    const handleTabChange = (tab: 'added' | 'removed') => {
+        setActiveTab(tab);
+    };
+
+    const displayName = channelName || channelId;
+
+    return (
+        <Modal
+            className='UserListModal'
+            dialogClassName='user-list-modal-dialog'
+            onExited={onClose}
+            show={true}
+            onHide={onClose}
+        >
+            <Modal.Header closeButton={true}>
+                <Modal.Title>
+                    <FormattedMessage
+                        id='admin.jobTable.syncResults.userListTitle'
+                        defaultMessage='Channel Members Changes - {channelName}'
+                        values={{
+                            channelName: displayName,
+                        }}
+                    />
+                </Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                <div className='tabs'>
+                    <button
+                        className={`tab-button ${activeTab === 'added' ? 'active' : ''}`}
+                        onClick={() => handleTabChange('added')}
+                    >
+                        <FormattedMessage
+                            id='admin.jobTable.syncResults.added'
+                            defaultMessage='Added ({count})'
```

Comment:
By default, FormatJS will render a 0 value as the empty string. Could you change these to tell it that these are numbers? This'll make it always render 0 properly:
```
defaultMessage='Added ({count, number})'
```